node/blockchain/tests/test_management/test_send_signed_change_request.py

```diff
@@ -64,26 +61,20 @@ def test_coin_transfer(
         stdout=out
     )
     _, output = out.getvalue().split('Response (raw):')
-    assert json.loads(output) == {
```

Comment:
this change reduces quality of the unittest, please, keep signature and signer assertion